src/SFA.DAS.EmployerIncentives.Web/Controllers/HubController.cs

```diff
@@ -40,9 +40,9 @@ public async Task<IActionResult> Index(string accountId, string accountLegalEnti
                 AccountLegalEntityId = accountLegalEntityId,
                 OrganisationName = selectedLegalEntity?.Name,
                 HasMultipleLegalEntities = legalEntities.Count() > 1,
-                ShowPhaseTwoClosureContent = ShowPhaseTwoClosureContent(_webConfiguration.ApplicationShutterPageDate)
+                ShowPhaseTwoClosureContent = ShowPhaseTwoClosureContent(_webConfiguration)
```

Comment:
This wouldn't achieve the effect of removing the banner, it would simply show the phase 2 content that shows the closing date for applications.
As the phase 3 feature branch is being deployed on the release date, it would be simpler to just remove the markup for the banner.

Reply:
The story is a bit unclear, but "no other existing banners are removed" means that the logic to show the "bank details required" and "legal agreement required" banners should still remain in place.

Reply:
The story doesn't cover not displaying the phase2 content.  I raised this with Faz and there are other stories in the pipeline to cover this.
Is the phase 3 feature branch being deployed on the release date?  Not aware of that. 
Yes this story is just to remove the phase 3 info not any other existing banners.

Reply:
The content doc linked to the related design ticket (EI-1502) doesn't have any mention of the notification banner content being shown, which I interpreted as the notification banner should be removed completely.

Reply:
Updated